modules-magic-drc/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:20.04
```

Comment:
Is there a Docker image that would be suitable that we could use from the https://github.com/hdl organization?

Reply:
Advice @umarcor?

Reply:
> Is there a Docker image that would be suitable that we could use from the https://github.com/hdl organization?

What is the use case of this Dockerfile? Is it for internal testing purposes or is it to be published?

> I guess maybe this one, except it's not pushed anywhere. https://raw.githubusercontent.com/hdl/containers/main/centos-7/base.dockerfile

Check https://github.com/orgs/hdl/packages and https://hdl.github.io/containers/#_tools_and_images. All images are pushed to ghcr.io; only the Debian collection is mirrored to docker.io.
Nonetheless, I don't understand why you thought about using CentOS, given that this PR is using Ubuntu. It feels more natural switching to Debian, unless you have some issue with Buster.

Overall, I think the sensible approach would be to add Magic to hdl/containers. So, we'd have hdlc/magic (or ghcr.io/hdl/debian-buster/magic, or ghcr.io/hdl/centos-7/magic). Then, this repo/PR might just add the python scripts on top of that.